modules/search/instant-search/components/jetpack-colophon.scss

```diff
@@ -0,0 +1,17 @@
+.jetpack-instant-search__jetpack-colophon {
```

Comment:
To prevent weirdness in some themes (see Lyrical below) we should also add:

`align-items: center;`


![image](https://user-images.githubusercontent.com/390760/73454127-58df2b80-4365-11ea-92da-19cb1b2c64fb.png)